.editorconfig

```diff
@@ -0,0 +1,18 @@
+# WordPress Coding Standards
+# https://make.wordpress.org/core/handbook/coding-standards/
+	
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+indent_style = tab
+
+[{.jshintrc,*.json,*.yml}]
```

Comment:
@westonruter I guess it is a bit rare to find JSON files written manually, maybe that was the rational behind this. I just took it off [wp-cli] (https://github.com/wp-cli/wp-cli/pull/2395/files) and I kinda support it. If JSON files are normally indented with 2 spaces by tools, then maybe they should all be the same ?

Couldn't find a clear instruction on WPCS as well.